src/h5web/toolbar/controls/DomainSlider.tsx

```diff
@@ -50,11 +53,17 @@ function DomainSlider(props: Props): ReactElement {
 
       <ToggleBtn
         small
-        label="Auto"
+        label={`Auto${
+          value[0] === undefined && value[1] !== undefined
+            ? ' (min)'
+            : value[0] !== undefined && value[1] === undefined
+            ? ' (max)'
+            : ''
+        }`}
```

Comment:
According to my comment above, this would be:
```js
       label={`Auto${
          isAutoMin
            ? ' (min)'
            : isAutoMax
            ? ' (max)'
            : ''
        }`}
```